docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -44,10 +44,8 @@ instead of ``no``:
     enable=yes
 
 Now when you run a workload, a trace output file will be placed in the
-``sim_slot_<slot #>`` directory on the F1 instance under the name ``TRACEFILE0-C0``.
-The first ``0`` in this filename disambiguates between multiple SoCs on one FPGA
-if you're running in supernode mode and will always be ``0`` if you're not running
-in supernode mode. The ``C0`` represents core 0 in the simulated
+``sim_slot_<slot #>`` directory on the F1 instance under the name ``TRACEFILE-C0``.
```

Comment:
Supernode support was deprecated?

Reply:
I believe they're just numbered 0..15 now, in a quad-core 4xsupernode system

Reply:
one option we have is to have the user specify part of file name when they instantiate the target-side of the bridge, since they have the module path information required to disambiguate between the 16 instances.

In supernoded configurations we can just make that be "node<n>" or just "" in there's only one.

The file could then be named <prefix>node<n>-C<i>